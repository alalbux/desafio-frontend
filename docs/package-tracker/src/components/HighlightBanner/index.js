import React from 'react';
import Grid from '@mui/material/Grid';
import Typography from '@mui/material/Typography';
import SearchPackage from '@/components/SearchPackage';
import LOCALES from '@/locales/hightLightBanner';

const HighlightBanner = () => {
  return (
    <Grid container justifyContent="space-evenly" sx={styles.container}>
      <Grid item xs={12} sm={4} sx={styles.textContainer}>
        <Typography variant="h3" sx={styles.text}>
          {LOCALES.TITLE}
        </Typography>
        <Typography variant="h5" sx={styles.text}>
          {LOCALES.SUBTITLE}
        </Typography>
      </Grid>
      <Grid item xs={12} sm={4}>
        <SearchPackage />
      </Grid>
    </Grid>
  );
};

const styles = {
  container: {
    pt: 10,
    pb: 6,
  },
  textContainer: {
    pt: 8,
    pb: 6,
  },
  text: {
    mb: 2,
    color: 'white',
    align: 'center',
  },
};

export default HighlightBanner;
