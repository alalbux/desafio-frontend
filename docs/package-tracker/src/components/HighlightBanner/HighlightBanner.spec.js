import React from 'react';
import { render } from '@testing-library/react';
import HighlightBanner from '@/components/HighlightBanner';

describe('HighlightBanner', () => {
  it('matches the snapshot', () => {
    const { container } = render(<HighlightBanner />);
    expect(container).toMatchSnapshot();
  });
});
