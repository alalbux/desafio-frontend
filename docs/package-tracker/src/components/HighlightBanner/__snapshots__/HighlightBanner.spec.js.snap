// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightBanner matches the snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container css-1oz2sp6-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-1nkx0c4-MuiGrid-root"
    >
      <h3
        class="MuiTypography-root MuiTypography-h3 css-109pgbq-MuiTypography-root"
      >
        A melhor solu칞칚o para envio de encomendas no Brasil.
      </h3>
      <h5
        class="MuiTypography-root MuiTypography-h5 css-5ndcwm-MuiTypography-root"
      >
        Alcance mais clientes com entregas locais e nacionais. Com a Loggi, voc칡 economiza tempo e dinheiro.
      </h5>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-wg4xta-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-19sicoy-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 css-1idn90j-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-7q1i3q-MuiPaper-root-MuiCard-root"
          >
            <div
              class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6 css-gof9t7-MuiTypography-root"
              >
                Ol치! Acompanhe com a Loggi a entrega do seu pedido. 游땎
              </h6>
              <p
                class="MuiTypography-root MuiTypography-body1 css-y3ensx-MuiTypography-root"
              >
                Primeiro, digite o c칩digo de rastreamento.
              </p>
            </div>
            <div
              class="MuiCardActions-root MuiCardActions-spacing css-s90r5k-MuiCardActions-root"
            >
              <form
                style="width: 100%; display: flex; flex-direction: column; justify-content: center; align-items: center;"
              >
                <div
                  class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-1d4eon-MuiFormControl-root-MuiTextField-root"
                >
                  <label
                    class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="false"
                    for="tracking-code-input"
                    id="tracking-code-input-label"
                  >
                    C칩digo de rastreamento
                  </label>
                  <div
                    class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                      id="tracking-code-input"
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        class="css-yjsfm1"
                      >
                        <span>
                          C칩digo de rastreamento
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-1tr9bli-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="submit"
                >
                  Acompanhar pedido
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
